Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -22,13 +40,20 @@ NOTE: Assigning an issue to Triage using the [Triage Incident Template](https://
 
 * Bugs that belong within your own team
 * Feature Improvements that belong within your own team
-* What are we missing here?
 
 ## Still not sure?
 If you still have a doubt about where to report your incident for whatever reason, please reach out to the #vsp-triage slack channel and we would be happy to assist you.
 
 When in doubt, submit any issue through our [Triage Incident Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=triage%2C+triage-incident&template=triage-incident-template.md&title=) and the Triage team will ensure it gets through the process correctly!
 
+
+
+
+
+
+
+# I Don't think we need to define this anymore?
```

Comment:
:+1: I think you pretty much did that in-line, yeah.

Reply:
that's a lot of extra whitespace